orchestrator-salt/src/main/resources/salt/salt/sizes/init.sls

```diff
@@ -0,0 +1,19 @@
+/opt/salt/scripts/get_backup_data_sizes.sh:
```

Comment:
Can we have a more related name and location for this?
From the context it seems it is only related to backup, so having a backup state and within that a sizes subtype would be more straightforward.

Reply:
I mean't `sizes` is too common and the entire state is about backup_data size, so I would suggest to have a backup/sizes.sls which makes it more clear what it does by it naming and location.

Reply:
I agree, it should be something like backup_size_calculation_files to be more expressive